0.3.0 breaks director service

The director service in 0.3.0 fails to run with the following error:

```sh
director_1   | > @sorry-cypress/director@0.3.0 start /app
director_1   | > node ./dist/
director_1   |
director_1   | internal/modules/cjs/loader.js:638
director_1   |     throw err;
director_1   |     ^
director_1   |
director_1   | Error: Cannot find module 'core-js/modules/es.promise'
director_1   |     at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
director_1   |     at Function.Module._load (internal/modules/cjs/loader.js:562:25)
director_1   |     at Module.require (internal/modules/cjs/loader.js:692:17)
director_1   |     at require (internal/modules/cjs/helpers.js:25:18)
director_1   |     at Object.<anonymous> (/app/dist/index.js:3:1)
director_1   |     at Module._compile (internal/modules/cjs/loader.js:778:30)
director_1   |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
director_1   |     at Module.load (internal/modules/cjs/loader.js:653:32)
director_1   |     at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
director_1   |     at Function.Module._load (internal/modules/cjs/loader.js:585:3)
```

--- a/packages/director/.babelrc
+++ b/packages/director/.babelrc
@@ -5,9 +5,7 @@
       {
         "targets": {
           "node": 10
-        },
-        "useBuiltIns": "usage",
-        "corejs": 3
+        }
       }
     ],
     "@babel/typescript"

